css/examples.css

```diff
@@ -0,0 +1,181 @@
+.examples__card {
+    position: relative;
+
+    width: 280px;
+    height: 120px;
+    margin: 0 auto;
+    margin-bottom: 10px;
```

Comment:
Use margin shortcut above to set `margin-bottom`

Reply:
done